### PR TITLE
Add exit command to command summary and fix add command params order

### LINKED ISSUE #253: Exit command is not present in command summary

### LINKED ISSUE #254: Add command field order in command summary is not consistent with field order in main UG
Under heading "Add Client Details: `add`":

Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS d/DEADLINE [t/TAG]…​
[ps/PROJECT_STATUS] [py/PAYMENT_STATUS] [cs/CLIENT_STATUS]`

In Command Summary table:

`add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​ [ps/PROJECT_STATUS] [py/PAYMENT_STATUS] [cs/CLIENT_STATUS] d/DEADLINE` <br> e.g., `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 t/friends ps/in progress py/pending cs/active d/10-11-2024`

Note that the placement of `d/DEADLINE` is different.

### PR DESCRIPTION
**Pull request by Dinoman44**
Fixes #253 
Fixes #254

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -341,9 +341,14 @@ Format: `archive-list`
...
-The archive list is a purely read-only list. You can only manipulate it using the command `unarchive`.
+The archive list is a purely read-only list. You can only use 4 commands while viewing it:
+
+1. `unarchive` (to unarchive a client)
+2. `list` (to navigate back to the main client list)
+3. `exit` (to exit the application)
+4. `archive-list` (redundant)
 <br>
-Other commands will require you to navigate back to the main client list first using the `list` command.
+All other commands will require you to navigate back to the main client list first using the `list` command.
```
Comment (Dinoman44) [inline docs/UserGuide.md:351]: Same as previous PR #251 my branches are still messed up

Comment (JunLongling) [reply docs/UserGuide.md:351]: Maybe you could add 'help' as well, since you decided to list all available commands.

Comment (Dinoman44) [reply docs/UserGuide.md:351]: Yep done, good catch

**Review by rahula1008 — APPROVED** [review]
LGTM

**Pull request merged**